enterprise/security/src/main/java/org/neo4j/server/security/enterprise/configuration/SecuritySettings.java

```diff
@@ -234,7 +254,10 @@
             ( logs ) -> new File( logs, "security.log" ),
             PATH );
 
-    @Description( "Set to log successful authentication events." )
+    @Description( "Set to log successful authentication events to the security log." +
+                  "If this is set to `false` only failed authentication events will be logged, which " +
+                  "could be useful if you find that the successful events spam the logs too much, " +
+                  "and you do not require full auditing capability")
```

Comment:
Code style *ought* to be: "One sentence per line." when writing docs, even if the docs happen to be embedded in other code. Not holding my breath about that one, though.